recipes/whatsapp/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "whatsapp",
   "name": "WhatsApp",
-  "version": "3.4.5",
+  "version": "3.4.4",
```

Comment:
you will need to bump up the version for this to flow to the user's machines @SpecialAro

Reply:
Yes! I noticed that mistake once I merged it.. Force pushed develop branch to the initial state and raised a new PR https://github.com/ferdium/ferdium-recipes/pull/171